conjur/cli.py

```diff
@@ -59,143 +63,154 @@ def copyright():
         global_optional.add_argument('-v', '--version', action='version',
                                      help="Display version number",
-                                     version='Conjur Python CLI version ' + __version__ + "\n"
+                                     version='Conjur CLI version ' + __version__ + "\n"
                                              + self.copyright())
 
-        global_optional.add_argument('-l', '--url')
-        global_optional.add_argument('-a', '--account')
-
-        global_optional.add_argument('-c', '--ca-bundle')
-        global_optional.add_argument('--insecure',
-            help='Skip verification of server certificate (not recommended)',
-            dest='ssl_verify',
-            action='store_false')
-
-        # The external names for these are unfortunately named so we remap them
-        global_optional.add_argument('-u', '--user', dest='login_id')
-        global_optional.add_argument('-k', '--api-key')
-        global_optional.add_argument('-p', '--password')
-
         global_optional.add_argument('-d', '--debug',
-            help='Enable debugging output',
-            action='store_true')
-
-        global_optional.add_argument('--verbose',
-            help='Enable verbose debugging output',
-            action='store_true')
+                                     help='Enable debugging output',
+                                     action='store_true')
 
+        global_optional.add_argument('--insecure',
+                                     help='Skip verification of server certificate (not recommended for production)',
+                                     dest='ssl_verify',
+                                     action='store_false')
 
         resource, args = Cli._parse_args(parser)
 
-        # We don't have a good "debug" vs "verbose" separation right now
-        if args.verbose is True:
-            args.debug = True
-
         # TODO Add tests for exception handling logic
+        # # pylint: disable=broad-except
         try:
+            self._setup_logging(args.debug)
+
             Cli.run_client_action(resource, args)
         except FileNotFoundError as not_found_error:
             sys.stdout.write(f"Error: No such file or directory: '{not_found_error.filename}'")
-            logging.info('Error: No such file or directory: %s', not_found_error.filename)
-            sys.exit(1)
         except requests.exceptions.HTTPError as http_error:
             sys.stdout.write(str(http_error))
-            logging.info(str(http_error))
-            sys.exit(1)
-        else:
-            sys.exit(0)
+        except ValueError as value_error:
+            sys.stdout.write(str(value_error))
+        except RuntimeError as runtime_error:
+            sys.stdout.write(str(runtime_error))
+        except Exception as general_exception:
+            sys.stdout.write(str(general_exception))
```

Comment:
If you're doing the same action for all 3 types of exceptions, there's no need for a triple separate case statement as it's all just the same `Exception` clause catch.

Reply:
What do you mean? I am catching very different types of exceptions (runtime, valueerror, and the more general exception)
Are you saying to create a function to handle the writing and exit of each exception?

Reply:
You can either:
`except Exception as e:` - and delete the `ValueError`, `RuntimeError` and `SSLError` clauses.
Or:
`except (ValueError, RuntimeError, SSLError) as e:` - and handle the exception different than the way we handle `Exception`